twitter_delete.rb

```diff
@@ -20,6 +20,7 @@
   opt :user, "The Twitter username to purge", type: :string, default: ENV["TWITTER_USER"]
   opt :csv, "Twitter archive tweets.csv file", type: :string
   opt :days, "Keep tweets under this many days old", default: 28
+  opt :olds, "Keep tweets more than this many days old", default: 1460
```

Comment:
I'd like the default here to remain the same i.e. `0` which is special-cased to mean delete tweets with no maximum age. Alternatively, set the default to a ludicrously high value like 50 years ago.